docs/checklist_applicationclass_2-markdown_v1.1.md

```diff
@@ -0,0 +1,157 @@
+
+**EQA.2** - **[ok]** The software responsible knows how to request specific support at the beginning and during development as well as to exchange ideas with other colleagues on the subject of software development. *(from application class 1)*
+
+**EQA.3** - **[ok]** The persons involved in the development determine the skills needed with regard to their role and the intended application class. They communicate these needs to the supervisor. *(from application class 1)*
+
+**EQA.4** - **[ok]** The persons involved in the development are given the tools needed for their tasks and are trained in their use. *(from application class 1)*
+
+## Requirements Management <a name="anforderungsmanagement"></a>
+
+:information_source: _For now, all of these are marked `[n.a.]` as this is usually done on a per-project basis in which CosmoScout VR is applied._ 
+
+**EAM.1** - **[n.a.]** The problem definition is coordinated with all parties involved and documented. It describes the objectives, the purpose of the software, the essential requirements and the desired application class in a concise, understandable way. *(from application class 1)*
+
+**EAM.2** - **[n.a.]** Functional requirements are documented at least including a unique identifier, a description, the priority, the origin and the contact person. *(from application class 2)*
+
+**EAM.3** - **[n.a.]** The constraints are documented. *(from application class 1)*
+
+**EAM.4** - **[n.a.]** The quality requirements are documented and prioritised. *(from application class 2)*
+
+**EAM.5** - **[n.a.]** User groups and their tasks are documented in the respective usage context. *(from application class 2)*
+
+**EAM.8** - **[n.a.]** A glossary exists which describes the essential terms and definitions. *(from application class 2)*
+
+**EAM.9** - **[n.a.]** The list of requirements is regularly coordinated, updated, analysed and checked. The resulting changes are traceable. *(from application class 2)*
+
+## Software Architecture <a name="software-architektur"></a>
+**ESA.1** - **[ok]** The architecture documentation is comprehensible for the relevant target groups. *(from application class 2)*
+
+**ESA.2** - **[ok]** Essential architectural concepts and corresponding decisions are at least documented in a lean way. *(from application class 1)*
+
+**ESA.3** - **[todo]** Testability of the software is appropriately addressed at software architecture level. *(from application class 2)*
+
+**ESA.4** - **[ok]** The software architecture is coordinated with the relevant target groups. Changes are communicated actively and are comprehensible. *(from application class 2)*
+
+**ESA.5** - **[ok]** The overlap between architectural documentation and implementation is minimised. *(from application class 2)*
+
+**ESA.6** - **[n.a.]** The architecture documentation consistently uses the terminology of the requirements. *(from application class 2)*
```

Comment:
Should be re-checked when high-level requirements are documented

Reply:
```suggestion
**ESA.6** - **[todo]** The architecture documentation consistently uses the terminology of the requirements. *(from application class 2)*
```